docs/dac/go/README.md

```diff
@@ -12,18 +12,6 @@ See the dedicated pages for each builder:
 - [Datasource](./datasource.md)
     - [HTTP Proxy](./helper/http-proxy.md)
 - [Panel](./panel.md)
-    - [Bar](./panel/bar.md)
```

Comment:
I think we should add a sentence in this file like you did in other places to mention plugins are documented elsewhere

Reply:
I think that's not necessary as it mentioned in many others places already.